Mathlib/SetTheory/Ordinal/FixedPointApproximants.lean

```diff
@@ -67,7 +67,7 @@ variable [CompleteLattice α] (f : α →o α) (x : α)
 open Function fixedPoints Cardinal Order OrderHom
 
 set_option linter.unusedVariables false in
-/-- Ordinal approximants of the least fixed point greater then an initial value x -/
+/-- The ordinal-indexed approximantion of the least fixed point greater than an initial value x -/
```

Comment:
I'd write down the recursive definition here in plaintext, something like
```text
The ordinal-indexed sequence of fixed point approximations `f^[a] x` is defined so that `f^[0] x = x` and `f^[a] x = ⨆ b < a, f (f^[a] x)`.
```
Also, you wrote down "approximantion" instead of "approximation".

Reply:
I decided against `f^[a]` as (1) mathlib only offers this notation for natural numbers, (2) we would have this notation twice in the file with different semantics - which seems confusing to me.

Otherwise, I split your suggestion into two sentences, so that it is hopefully a bit nicer to read.

Reply:
Yeah, this new docstring is much better.